services/api/kalvi/middlewares/tokenMiddleware.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework_simplejwt.tokens import AccessToken
+from rest_framework import status
+from kalvi.api.views.auth import get_redis_connection
+from django.http import JsonResponse
+
+class CustomOutstandingTokenMiddleware:
+    """
+    This middleware checks for blacklisted tokens in Redis. It does not perform any token validation.
+    """
+    def __init__(self, get_response=None):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        try:
+            auth_header = request.headers.get('Authorization')
+            if not auth_header:
+                return self.get_response(request)
+            token = auth_header.split()[1]
+            token_obj = str(AccessToken(token))
+            redis_conn = get_redis_connection()
```

Comment:
Create a separate class for Redis in a separate file and implement the logic there.

Reply:
there should be separate file for redis config too and blacklisting refresh token too, I am thinking raising seperate PR for this , kinda chore/refactor tasks. I will put everything related to redis at one place.

Reply:
what is your view on this?